pkgs/development/interpreters/python/update-python-libraries/update-python-libraries.py

```diff
@@ -232,6 +232,7 @@ def get_prefix(string):
 
 FETCHERS = {
     'fetchFromGitHub'   :   _get_latest_version_github,
+    'fetchFromPyPI'         :   _get_latest_version_pypi,
```

Comment:
alignment looked off from the github ui
```suggestion
    'fetchFromGitHub'   :   _get_latest_version_github,
    'fetchFromPyPI'     :   _get_latest_version_pypi,
```